cli/cmdlineparser.cpp

```diff
@@ -949,13 +948,15 @@ void CmdLineParser::PrintHelp()
               "                                 32 bit Windows UNICODE character encoding\n"
               "                          * win64\n"
               "                                 64 bit Windows\n"
-              "                          * avr8\n"
-              "                                 8 bit AVR microcontrollers\n"
               "                          * native\n"
               "                                 Type sizes of host system are assumed, but no\n"
-              "                                 further assumptions.\n"
+              "                                 further assumptions\n"
               "                          * unspecified\n"
               "                                 Unknown type sizes\n"
+              "                         --platform=<file>\n"
+              "                         Uses types and sizes from platform file:\n"
+              "                          * platforms/avr8.xml\n"
```

Comment:
If the stuff is in a separate file, I would recommend to not mention it in the --help output.

Reply:
Why not? What's the difference between library and platform files?

Reply:
in the help for --library we don't list what cfg files there are.